py_gtfs_rt_ingestion/lib/s3_utils.py

```diff
@@ -152,17 +134,35 @@ def _move_s3_object(filename: str, destination: str) -> None:
         process_logger.log_failure(exception)
 
 
-def move_s3_objects(files: List[str], destination: str) -> None:
+def _init_process_session() -> None:
     """
-    Move list of S3 objects from source to destination.
+    initialization function for any process in multi processing pool needing to
+    use a boto session object
+
+    this avoids the expensive operation of creating a new session for every unti of work
+    in the pool
+
+    not totally sure this is the best way to retain the session on process initialization
+    but it seems to work
```

Comment:
i guess this session is only going to live as long as we're moving a list of files, so this should be fine.